src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -73,7 +73,8 @@ def load_arguments(self, _):
         c.ignore('max_burst')
 
     with self.argument_context('appservice plan create') as c:
-        c.argument('name', options_list=['--name', '-n'], help="Name of the new app service plan", completer=None)
+        c.argument('name', options_list=['--name', '-n'], help="Name of the new app service plan", completer=None,
+                   validator=validate_asp_create)
```

Comment:
For future reference, if you want a specific command to have custom validation, you can put a validator on the command registration instead of the parameters. Don't change for this PR because it would likely introduce more work, but if you do look in network at the "process_foo_create_namespace" style validators you'll see how they are used.

Reply:
Got it. Thanks for the suggestion will remember it for future reference.